bioblend/galaxy/genomes/__init__.py

```diff
@@ -51,17 +63,17 @@ def show_genome(self, id, num=None, chrom=None, low=None, high=None):
             params["low"] = low
         if high:
             params["high"] = high
-        return self._get(id, params)
+        return self._get(id=id, params=params)
```

Comment:
Thanks for the fix, that was broken!

Reply:
yeap, thanks to Fabio! it kept asking me for a boolean here, and it didn't make any sense to me.

Reply:
Group work :heart: